tray/actions.go

```diff
@@ -75,6 +77,17 @@ func (ti *Instance) logout(persistToken bool) bool {
 	}
 }
 
+func (ti *Instance) notifyServiceExpired(url string, trustedPassURL string, message string) {
+	resp, err := ti.client.TokenInfo(context.Background(), &pb.Empty{})
+	isTokenDataValid := resp.TrustedPassToken != "" && resp.TrustedPassOwnerId != ""
```

Comment:
if there is an error, it might be good not to use `resp` because it might be `nil`